src/mam4xx/convproc.hpp

```diff
@@ -263,6 +263,56 @@ void update_tendency_final(
   }
 }
 // =========================================================================================
+// clang-format off
+// nlev = number of atmospheric levels: 0 <= ktop <= kbot_prevap <= nvel
+// nlevp = nlev + 1
+KOKKOS_INLINE_FUNCTION
+void compute_column_tendency(
+  const int ktop,                          // top level index
+  const int kbot_prevap,                   // bottom level index, for resuspension and evaporation only
+  const bool doconvproc_extd[ConvProc::pcnst_extd],  // IN flag for doing convective transport
+  const Real dpdry_i[/*nlev*/],                      // IN dp [mb]
+  const Real dcondt_resusp[/*nlev*/][ConvProc::pcnst_extd],    // IN portion of TMR tendency due to resuspension [kg/kg/s]
+  const Real dcondt_prevap[/*nlev*/][ConvProc::pcnst_extd],    // IN portion of TMR tendency due to precip evaporation [kg/kg/s]
+  const Real dcondt_prevap_hist[/*nlev*/][ConvProc::pcnst_extd], // IN  of TMR tendency due to precip evaporation, goes into the history [kg/kg/s]    
```

Comment:
looks like there might be a missing word here and missing `d`s in the following two lines.